infra/terraform/modules/lambda/variables.tf

```diff
@@ -1,34 +1,80 @@
 variable "REGION" {
-  type    = string
-  default = "eu-west-1"
+  type        = string
+  description = "The region where the module is deployed"
+  default     = "eu-west-1"
 }
 
 variable "ENV" {
-  type    = string
-  default = "dev"
+  type        = string
+  description = "The environment where the lambda will be executed."
+  default     = "dev"
 }
 
 variable "PROJECT_NAME" {
-  type    = string
-  default = "tezos-link"
+  type        = string
+  description = "The name of the project associated to the lambda."
+  default     = "tezos-link"
 }
 
 variable "BUILD_WITH" {
-  type    = string
-  default = "terraform"
+  type        = string
+  description = "Permits to know on the AWS tags that objects are build with IaC."
+  default     = "terraform"
 }
 
-variable "SNAPSHOT_S3_KEY" {
-  type    = string
-  default = "v1.0.0/snapshot.zip"
+variable "VPC_CIDR" {
+  type        = string
+  description = "The CIDR of the VPC where the lambda will be placed"
+  default     = ""
 }
 
-variable "NODE_USER" {
-  type    = string
-  default = "ec2-user"
+variable "S3_BUCKET_NAME" {
+  type        = string
+  description = "The S3 bucket name that will be associated to the lambda"
 }
 
-variable "S3_LAMBDA_KEY" {
-  type    = string
-  default = "snapshot_lambda_key"
+variable "S3_CODE_PATH" {
+  type        = string
+  description = "The path where the code is stored."
+  default     = "v1.0.0/snapshot.zip"
 }
+
+variable "LAMBDA_PURPOSE" {
```

Comment:
i don't think PURPOSE is a sensible name as it lead to think that this variable is similar to the LAMBDA_DESCRIPTION variable.

Reply:
Hum ok. I understand. Do you have an idea on how we could call it ? `LAMBDA_NAME` ?

Reply:
yes i think it a more intelligible name

Reply:
Done :) Thanks for the review.

Reply:
both are ok for me